### PR TITLE
fix: vite build binding not found error

### LINKED ISSUE #89: Getting a lot of errors when following the contribution documentation
Cloning a fresh repo from master yields the following errors following the steps provided here: [git@github.com:shannonhochkins/ha-component-kit.git](https://github.com/shannonhochkins/ha-component-kit/blob/master/CONTRIBUTING.md#local-development--contribution)

Exact repro steps:
1. Clone repository
2. `npm install`
3. `npm start`
4. Notice the following errors:
```bash
[vite] Internal server error: Failed to resolve entry for package "@hakit/core". The package may have incorrect main/module/exports specified in its package.json.
  Plugin: vite:import-analysis
  File: /home/yann510/github/test/ha-component-kit/packages/components/src/Shared/ColorPicker/index.tsx
      at packageEntryFailure (file:///home/yann510/github/test/ha-component-kit/node_modules/vite/dist/node/chunks/dep-df561101.js:28691:11)
      at resolvePackageEntry (file:///home/yann510/github/test/ha-component-kit/node_modules/vite/dist/node/chunks/dep-df561101.js:28688:5)
      at tryNodeResolve (file:///home/yann510/github/test/ha-component-kit/node_modules/vite/dist/node/chunks/dep-df561101.js:28419:20)
      at Context.resolveId (file:///home/yann510/github/test/ha-component-kit/node_modules/vite/dist/node/chunks/dep-df561101.js:28180:28)
      at runMicrotasks (<anonymous>)
      at processTicksAndRejections (node:internal/process/task_queues:96:5)
      at async Object.resolveId (file:///home/yann510/github/test/ha-component-kit/node_modules/vite/dist/node/chunks/dep-df561101.js:44207:32)
      at async TransformContext.resolve (file:///home/yann510/github/test/ha-component-kit/node_modules/vite/dist/node/chunks/dep-df561101.js:43923:23)
      at async normalizeUrl (file:///home/yann510/github/test/ha-component-kit/node_modules/vite/dist/node/chunks/dep-df561101.js:41793:34)
      at async file:///home/yann510/github/test/ha-component-kit/node_modules/vite/dist/node/chunks/dep-df561101.js:41945:47
      at async Promise.all (index 7)
      at async TransformContext.transform (file:///home/yann510/github/test/ha-component-kit/node_modules/vite/dist/node/chunks/dep-df561101.js:41870:13)
      at async Object.transform (file:///home/yann510/github/test/ha-component-kit/node_modules/vite/dist/node/chunks/dep-df561101.js:44283:30)
      at async loadAndTransform (file:///home/yann510/github/test/ha-component-kit/node_modules/vite/dist/node/chunks/dep-df561101.js:54950:29)
      at async viteTransformMiddleware (file:///home/yann510/github/test/ha-component-kit/node_modules/vite/dist/node/chunks/dep-df561101.js:64345:32)
```

### PR DESCRIPTION
**Pull request by yann510**
closes #89

### CODE REVIEW

**Review by shannonhochkins — COMMENTED** [review]
Also a bit concerned why your lock file is so different after installing a few packages

File: packages/create-hakit/package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
   },
   "engines": {
     "node": ">=16.0.0",
-    "npm": "^7.0.0"
+    "npm": ">=7.0.0"
```
[the remark this answers is not in the document]
Comment (yann510) [reply packages/create-hakit/package.json:27]: We should probably build and run the tests in the CI to ensure pull requests are working at a minimum level

[the remark this answers is not in the document]
Comment (shannonhochkins) [reply packages/create-hakit/package.json:27]: Tests have been broken for a while, I decided to park tests while the architecture was changing so much 🙃 all good I'd say the biggest descrepency here with the lock file, is that you most likely followed step 1 in the link I sent? If you had removed the lock file then run npm I it explains the diff here :)

[the remark this answers is not in the document]
Comment (yann510) [reply packages/create-hakit/package.json:27]: Yeah I added some new dev packages to ensure it works properly for everyone.
I don't think a big change in the lock file is an issue, it is actually expected when you add new packages :)

I've been testing with linked package on my test dashboard and everything seems to be working just fine.

Need anything changed before we merge this?

[the remark this answers is not in the document]
Comment (shannonhochkins) [reply packages/create-hakit/package.json:27]: Sweet, and nope no changes, just need some time to check it out and test it on my machine to ensure it works for me 😅

[the remark this answers is not in the document]
Comment (yann510) [reply packages/create-hakit/package.json:27]: Awesome, keep me posted :)

**Review by shannonhochkins — APPROVED** [review]

**Pull request merged**